dataplane/cmd/sdx.go

```diff
@@ -315,6 +315,18 @@ func init() {
 					}
 				},
 			},
+			{
+				Name:   "sync-component-versions",
+				Usage:  "sync SDX cluster component version from CM",
```

Comment:
nit: the message could be similar to that of DistroX:
synchronizes component versions from CM for a SDX cluster